example/advect/t8_advection.cxx

```diff
@@ -452,9 +449,11 @@ t8_advect_flux_upwind_hanging (const t8_advect_problem_t *problem, t8_locidx_t i
     neigh_data = (t8_advect_element_data_t *) t8_sc_array_index_locidx (problem->element_data, neigh_id);
     neigh_is_ghost = neigh_id >= t8_forest_get_local_num_elements (problem->forest);
     phi_minus = t8_advect_element_get_phi (problem, neigh_id);
+    /* clang-format off */
     /* Compute the flux */
-    el_hang->fluxes[face][i]
-      = t8_advect_flux_upwind (problem, phi_plus, phi_minus, ltreeid, face_children[i], child_face);
+    el_hang->fluxes[face][i] = t8_advect_flux_upwind (problem, phi_plus, phi_minus, ltreeid,
+                                                      face_children[i], child_face);
+    /* clang-format on */
```

Comment:
```suggestion
```
The clang format will break befor "=", if the lign exceeds 120 columns. I think this is fine

Reply:
Please have another look at the PR and double-check the /* clang-format off/on */ encapsulations.